packages/f-checkout/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout – Fozzie Checkout Component",
-  "version": "0.14.0",
+  "version": "0.15.0",
```

Comment:
Do we have a precedent for minor version bumps when there are no functional code changes? (Wondering if this should be a patch, but don't let this block the PR)